front/admin/cli.ts

```diff
@@ -560,6 +567,24 @@ const transcripts = async (command: string, args: parseArgs.ParsedArgs) => {
   }
 };
 
+const registry = async (command: string, args: parseArgs.ParsedArgs) => {
+  switch (command) {
+    case "dump": {
+      Object.keys(DustProdActionRegistry).forEach((key) => {
```

Comment:
Let's add a comment to prevent someone from breaking the format below 🙏

Reply:
Or if we can use `jq`

Reply:
let's use jq